utils/pps-tools/patches/001-time_t_64bit.patch

```diff
@@ -0,0 +1,31 @@
+--- a/ppstest.c
++++ b/ppstest.c
```

Comment:
Did you send this patch to upstream or is this included in upstream already?

Reply:
it's not upstream, and i haven't opened a PR there yet. seems kind of inactive, though.

Reply:
Please open the PR there, otherwise, we would need to carry this forever, which is not good.

Reply:
https://github.com/redlab-i/pps-tools/pull/16